### DISCUSSION TITLE
请问改了logo后手机端不显示是怎么回事？

### DISCUSSION THREAD

**Discussion by sucooer** [imsyy/home#71] [General]
换了同尺寸的logo，pc端可以正常显示，但是到了手机端就无法显示
[![pS6LQ8H.png](https://s1.ax1x.com/2023/02/05/pS6LQ8H.png)](https://imgse.com/i/pS6LQ8H)

**Comment by imsyy** [comment]
手机端做了移动端适配，图片隐藏了，要不然太占位置

**Reply by sucooer** [reply]
有办法显示吗？左边空荡荡感觉怪怪的

**Reply by imsyy** [reply]
将来这个页面还会改，如果想调整，可以自行修改这里的 `css`
https://github.com/imsyy/home/blob/0c4a85b4586dd8f2fc020b8fa965d5c08dcfc79a/src/components/Message/index.vue#L156
